book/src/css.md

```diff
@@ -0,0 +1,266 @@
+
+<span class="filename">Filename: listings/css/7/style.css</span>
+
+```css
+{{#rustdoc_include ../listings/css/7/style.css}}
+```
+
+Indeed, we get a `ComboBoxText` with a red arrow.
+
+<div style="text-align:center"><img src="img/css_7.png"/></div>
+
+## Images
+
+Do we really want to settle with an arrow?
+Maybe there are much nicer icons out there you do not even know about yet.
+Fear not, the [Icon Library](https://apps.gnome.org/app/org.gnome.design.IconLibrary/) has you covered there.
+Scroll through the provided icons and choose one from the pre-installed system icons section.
+It's possible to embed icons from other sections too, we'll learn about that in a follow-up chapter.
+
+For now, we will take the icon with the semantic name "format-justify-fill-symbolic".
+In the screenshot below you see that this corresponds to three parallel lines.
+However, on a different system this icon might look different or not exist at all.
+This is why it is typically better to embed the icons you use.
+
+GTK provides its own parameter for loading icons in buttons and expanders: `-gtk-icon-source`.
+You can find the full list of supported parameters in GTK's [documentation](https://docs.gtk.org/gtk4/css-properties.html#gtk-css-properties).
+
+
+<span class="filename">Filename: listings/css/8/style.css</span>
+
+```css
+{{#rustdoc_include ../listings/css/8/style.css}}
+```
+
+Voilà, the arrow is replaced with the one we selected ourselves. 
+
+<div style="text-align:center"><img src="img/css_8.png"/></div>
+
+
+## Exported Colors
+
+Now that we know how to use CSS, it is time to update our To-Do app a bit.
+Before, the individual tasks were a bit hard to distinguish.
+Let us change that!
+
+First, we will add the style class `todo_row` to our `TodoRow`.
+
+<span class="filename">Filename: listings/todo_app/3/todo_row/todo_row.ui</span>
+
+```xml
+{{#rustdoc_include ../listings/todo_app/3/todo_row/todo_row.ui}}
+```
+
+One way to make the rows set apart a bit is to add borders.
+But which color to use for the borders?
+Our To-Do app should represent a real world app, so we will not be satisfied with choosing yellow and calling it a day.
+The stylesheet that GTK uses pre-defined colors for various use-cases, so let us have a look.
+As of this writing, the exported colors of the default stylesheet can be found in its [source code](https://gitlab.gnome.org/GNOME/gtk/-/blob/b2c227e9c57839a2a4e24462a71ae0bad9a95264/gtk/theme/Default/_colors-public.scss).
+
+There we find the color `borders`, which should be used for the widgets' border color.
+Perfect!
+We can access pre-defined colors by adding an `@` in front of their names.
+
+<span class="filename">Filename: listings/todo_app/3/style.css</span>
+
+```css
+{{#rustdoc_include ../listings/todo_app/3/style.css}}
+```
+
+Now our tasks have borders around them, and we are one step further in finishing our To-Do app.
+
+<div style="text-align:center"><img src="img/todo_app_change.png"/></div>
+
+## Conclusion
+
```

Comment:
Mention here a gain the complete CSS syntax `css-name#name.class-1.class-2`.
Otherwise, I expect that people will be confused about the difference between `css-name` and `name`.